activerecord/test/cases/relation/or_test.rb

```diff
@@ -31,7 +31,7 @@ def test_or_with_null_right
     end
 
     def test_or_with_bind_params
-      assert_equal Post.find([1, 2]), Post.where(id: 1).or(Post.where(id: 2)).to_a
+      assert_equal Post.find([10, 11]).sort, Post.where(id: 10).or(Post.where(id: 11)).to_a.sort
```

Comment:
How about adding `.sort_by(&:id)` simply?

```diff
--- a/activerecord/test/cases/relation/or_test.rb
+++ b/activerecord/test/cases/relation/or_test.rb
@@ -31,7 +31,7 @@ def test_or_with_null_right
     end
 
     def test_or_with_bind_params
-      assert_equal Post.find([1, 2]), Post.where(id: 1).or(Post.where(id: 2)).to_a
+      assert_equal Post.find([1, 2]).sort_by(&:id), Post.where(id: 1).or(Post.where(id: 2)).sort_by(&:id)
     end
 
     def test_or_with_null_both
```

Reply:
Thanks for the review. Much better one since no need to change id values.